src/demo.ts

```diff
@@ -0,0 +1,70 @@
+import { randomBytes } from "crypto"
+import { Client } from "./client"
+import { hrTime } from "./utils"
+
+/**
+ * Sends a demonstration/test sample for a exception and a performance issue.
+ *
+ * Note that the agent must be active for at least 60 seconds in order for the payload
+ * to be sent to AppSignal.
+ *
+ * This is also a good place to start if you want to see what custom instrumentation
+ * looks like in the wild!
```

Comment:
I'm not sure this comment is true anymore? We're using a function here that isn't meant to be used outside of the span processor.

We haven't really discussed what custom instrumentation post-OpenTelemetry is going to look like yet, but I don't think we'll have users call `extension.createOpenTelemetrySpan` directly. I think we've mentioned the possibility of exposing the OT tracer.

Reply:
I don't think it was ever true. I just copy-pasted it. I will remove it.
I don't know what we're going to do with the tracer.